pkg/validation/options.go

```diff
@@ -96,6 +96,8 @@ func Validate(o *options.Options) error {
 		})
 	}
 
+	msgs = append(msgs, validateApiRoutes(o)...)
```

Comment:
Can you move this up to the top of the function, I don't think we need to have it down here right?